benchmarker.py

```diff
@@ -102,7 +115,7 @@ def save_to_html(results):
 	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
 	html += "<tr>" + heading + "</tr>"
 	for i in range(len(languages)):
-		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
+		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str("{:.3f}".format(round(next, 3))) + " s" + "</td>" for next in results[i]])
```

Comment:
Sometimes, it finishes exactly like, for example, 0.06 so instead of printing out 0.060 s, it prints out 0.06 s .It's not big deal, but just aligning everything :) , what do you think?